starter/README.md

```diff
@@ -1,20 +1,18 @@
 # Base Starter for Vaadin Flow and OSGi in NPM mode
 
-Servlet initial parameters may be passed via `@Component` annotation parameter `property`, e.g.
+Vaadin Servlet initial parameters may be passed via `@Component` annotation parameter `property`, e.g.
```

Comment:
Servlet init parameter was the exact description.
But since this is about Vaadin it can be kept as is.